src/System.Security.Permissions/tests/SecurityManagerTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Security.Permissions.Tests
+{
+    public class SecurityManagerTests
+    {
+        [Fact]
+        public void GetStandardSandbox_WhenEvidenceIsNull_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() => SecurityManager.GetStandardSandbox(null));
```

Comment:
use AssertExtensions to also check for the name of the argument thrown (nameof(evidence))